pypeit/reduce.py

```diff
@@ -613,17 +731,21 @@ def find_objects_pypeline(self, image, std=False, ir_redux=False, std_trace=None
 
             sobjs_slit, skymask[thismask] = \
                 extract.objfind(image, thismask, self.tslits_dict['slit_left'][:,slit],
-                                self.tslits_dict['slit_righ'][:,slit], inmask=inmask, ir_redux=ir_redux,
-                                ncoeff=self.redux_par['trace_npoly'], std_trace=std_trace,
-                                sig_thresh=self.redux_par['sig_thresh'], hand_extract_dict=manual_extract_dict,
+                                self.tslits_dict['slit_righ'][:,slit], inmask=inmask,
+                                ir_redux=self.ir_redux,
+                                ncoeff=self.par['scienceimage']['findobj']['trace_npoly'],
+                                std_trace=std_trace,
+                                sig_thresh=self.par['scienceimage']['findobj']['sig_thresh'],
+                                hand_extract_dict=manual_extract_dict,
                                 specobj_dict=specobj_dict, show_peaks=show_peaks,
                                 show_fits=show_fits, show_trace=show_trace,
-                                trim_edg=self.redux_par['find_trim_edge'],
-                                cont_fit=self.redux_par['find_cont_fit'],
-                                npoly_cont=self.redux_par['find_npoly_cont'],
-                                fwhm=self.redux_par['find_fwhm'],
-                                maxdev=self.redux_par['find_maxdev'],
-                                qa_title=qa_title, nperslit=self.redux_par['maxnumber'], debug_all=debug)
+                                trim_edg=self.par['scienceimage']['findobj']['find_trim_edge'],
```

Comment:
Should this parset be renamed to reduce and not scienceimage??

Reply:
It makes sense to me, although there is already
a `ReducePar` class.  So that would need to be
renamed and then all existing `pypeit` files would
be wrong.

That is unless we maintain `rdx` doing what it does, etc.
Which would mean we are only making the changes internally
and we'd still use `scienceimage` externally.

Reply:
punting..

Reply:
How about ReduxPar and ReducePar where the former is what is currently Reduce par. That is more consistent with the naming which is rdx I believe.

Reply:
Done.